packages/terminal/src/tokens.ts

```diff
@@ -111,6 +111,11 @@ export namespace ITerminal {
      * Whether to auto-fit the terminal to its host element size.
      */
     autoFit?: boolean;
+
+    /**
+     * Treat option as meta key on macOS.
+     */
+    macOptionIsMeta: boolean;
```

Comment:
I think this key should be optional as well (`macOptionIsMeta?:`).

Reply:
Thanks, fixed.